README.md

```diff
@@ -1,2 +1,16 @@
-# docs
-Public documentation
+# Write code for the Internet Computer
```

Comment:
Dual purpose of the README. First as the entry to the actual documentation, second as the landing page for people who want to contribute to it.